Ast/src/Parser.cpp

```diff
@@ -2615,6 +2639,72 @@ AstExpr* Parser::parseString()
         return reportExprError(location, {}, "String literal contains malformed escape sequence");
 }
 
+AstExpr* Parser::parseInterpString()
```

Comment:
Btw, we'll want this parser to produce an AstExprConstantString if there's zero interpolating expressions. This makes it fold very nicely with everything else, plus is itself an optimization for the compiler so `` `foo` `` doesn't generate the same bytecode as `("foo"):format()`.

Reply:
Lexer now gives QuotedString when there are no expressions.

Reply:
Changed the lexer thing to something else, but this is still resolved